app/entrypoint.sh

```diff
@@ -1,3 +1,10 @@
 #!/bin/sh
 
-uwsgi /config/uwsgi.yaml
+if $APM_ENABLED; then
+    echo "APM tracing enabled: $APM_APP_NAME"
```

Comment:
Small typo here that was fixed in a later pr, APM_APP_NAME should be DATADOG_SERVICE_NAME.